packages/brand-positioning-block/src/utilities/settingsToCss.spec.ts

```diff
@@ -0,0 +1,156 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Mock, afterEach, beforeEach, describe, expect, test, vi } from 'vitest';
+import { borderSettingsToCss, itemStyleSettingssToCss, styleSettingsToCss } from './settingsToCss';
+import { BorderStyle, Radius } from '@frontify/guideline-blocks-shared';
+import { BrandItemSize } from '../types';
+
+const MOCK_RGBA_COLOR = 'rgba(0, 0, 0, 0.5)';
+const BACKGROUND_COLOR = { red: 225, green: 14, blue: 48, alpha: 0.8 };
+const BORDER_COLOR = { red: 12, green: 228, blue: 48, alpha: 0.8 };
+const MOCK_STYLE_SETTINGS = {
+    backgroundColor: BACKGROUND_COLOR,
+    borderColor: BORDER_COLOR,
+    borderWidth: '1px',
+    borderStyle: BorderStyle.Dotted,
+    borderRadius: Radius.None,
+    hasBorder: true,
+    hasCustomRadius: false,
+    customBorderRadius: '15px',
+};
+const RESULT_CSS = {
+    backgroundColor: MOCK_RGBA_COLOR,
+    border: `1px dotted ${MOCK_RGBA_COLOR}`,
+    borderRadius: '0px',
+};
+
+vi.mock('@frontify/guideline-blocks-shared', () => ({
+    borderStyleMap: {
+        ['Solid']: 'solid',
+        ['Dotted']: 'dotted',
+        ['Dashed']: 'dashed',
+    },
+    radiusStyleMap: {
+        ['None']: '0px',
+        ['Small']: '2px',
+        ['Medium']: '4px',
+        ['Large']: '12px',
+    },
+    toHex8String: vi.fn(),
+    BorderStyle: {
+        Solid: 'Solid',
+        Dashed: 'Dashed',
+        Dotted: 'Dotted',
+    },
+    Radius: {
+        None: 'None',
+        Small: 'Small',
+        Medium: 'Medium',
+        Large: 'Large',
+    },
+    BrandItemSize: {
+        S: 'Small',
+        M: 'Medium',
+        L: 'Large',
+    },
+    brandItemSizeMap: {
+        S: '40px',
+        M: '48px',
+        L: '56px',
+    },
+}));
+
+describe('borderSettingsToCss', () => {
+    let toHex8StringMock: Mock<any[], any>;
+    beforeEach(async () => {
+        const { toHex8String } = await import('@frontify/guideline-blocks-shared');
+        vi.mocked(toHex8String).mockReturnValue(MOCK_RGBA_COLOR);
+        toHex8StringMock = vi.mocked(toHex8String);
+    });
```

Comment:
I don't need the function, but importing from `@frontify/guideline-blocks-shared` - even in a dependency - will give me the following error: `Error: require() of ES Module /Users/jirimatyas/frontify/guideline-blocks/node_modules/.pnpm/lodash-es@4.17.21/node_modules/lodash-es/lodash.js from /Users/jirimatyas/frontify/guideline-blocks/node_modules/.pnpm/@frontify+app-bridge@3.0.0-beta.38_sfoxds7t5ydpegc3knd667wn6m/node_modules/@frontify/app-bridge/dist/index.umd.js not supported.
Instead change the require of lodash.js in /Users/jirimatyas/frontify/guideline-blocks/node_modules/.pnpm/@frontify+app-bridge@3.0.0-beta.38_sfoxds7t5ydpegc3knd667wn6m/node_modules/@frontify/app-bridge/dist/index.umd.js to a dynamic import() which is available in all CommonJS modules.`
Plus it's nice being able to verify that function is being called with the right argument, rather than just verifying the output.